readme.md

```diff
@@ -2,11 +2,12 @@
 
 The Jira/Mattermost plugin documentation is currently being updated and relocated to a new location: https://mattermost.gitbook.io/jira-plugin/ - let us know your thoughts on the new format in the [Plugin: Jira Channel](https://community-daily.mattermost.com/core/channels/jira-plugin) on our Mattermost community!
 
-
 # Mattermost Jira Plugin
 
-[![Build Status](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-jira/master.svg)](https://circleci.com/gh/mattermost/mattermost-plugin-jira)
-[![Code Coverage](https://img.shields.io/codecov/c/github/mattermost/mattermost-plugin-jira/master.svg)](https://codecov.io/gh/mattermost/mattermost-plugin-jira)
+[![Build Status](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-jira/master)](https://circleci.com/gh/mattermost/mattermost-plugin-jira)
+[![Code Coverage](https://img.shields.io/codecov/c/github/mattermost/mattermost-plugin-jira/master)](https://codecov.io/gh/mattermost/mattermost-plugin-jira)
+[![Release](https://img.shields.io/github/v/release/mattermost/mattermost-plugin-jira)](https://github.com/mattermost/mattermost-plugin-jira/releases)
+[![HW](https://img.shields.io/github/issues/mattermost/mattermost-plugin-jira/Up%20For%20Grabs?color=dark%20green&label=HW%20%28Up%20For%20Grabs%29)](https://github.com/mattermost/mattermost-plugin-jira/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Up+For+Grabs%22+label%3A%22Help+Wanted%22)
```

Comment:
`Help Wanted` is the most popular term for issue that should be worked on by the community. I'm 2/5 that it's the better term as it's easily understandable. 
```suggestion
[![HW](https://img.shields.io/github/issues/mattermost/mattermost-plugin-jira/Up%20For%20Grabs?color=dark%20green&label=Help%20Wanted)](https://github.com/mattermost/mattermost-plugin-jira/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Up+For+Grabs%22+label%3A%22Help+Wanted%22)
```

Reply:
I'm glad you brought this one up. As I currently have it, the link is to `HW` `UFG` tickets and the number shows the corresponding number of tickets..  
![image](https://user-images.githubusercontent.com/7575921/82938955-31d1e880-9f58-11ea-9438-359571e36a54.png)

I thought about Using only `Help Wanted` with the number of HW tickets, but that is kind of misleading, as I felt the number of HW UFG tickets should be advertised instead.

Reply:
While I see the point that some might be confused, it also streamlines the process for new community members a lot of they just have to care about UFG.   To me getting new community members to the right tickets is more important.